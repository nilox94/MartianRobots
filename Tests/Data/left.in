1 1
0 0 N
L
0 0 E
L
0 0 S
L
0 0 W
L
