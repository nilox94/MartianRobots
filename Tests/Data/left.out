0 0 W
0 0 N
0 0 E
0 0 S
